data/json/items/newspaper.json

```diff
@@ -592,5 +592,21 @@
     "price_postapoc": 0,
     "material": [ "paper" ],
     "flags": [ "TRADER_AVOID", "BANK_NOTE_SHAPED" ]
+  },
+  {
+    "type": "GENERIC",
+    "id": "home_picture",
+    "category": "books",
+    "symbol": "*",
+    "color": "white",
+    "name": { "str": "Picture" },
+    "snippet_category": "home_pictures",
+    "description": "A medium picture of something that the owner wished to decorate their house with.",
+    "weight": "120 mg",
+    "volume": "10 ml",
+    "price": 0,
+    "price_postapoc": 0,
+    "material": [ "paper", "wood" ],
+    "flags": [ "TRADER_AVOID", "BANK_NOTE_SHAPED" ]
```

Comment:
Bank_note_shaped would allow this item to be slipped within a wallet, which feels somewhat wrong if you’re intending the photograph and frame to already be one.